rmgpy/rmg/pdep.py

```diff
@@ -429,20 +429,30 @@ def solve_SS_network(self,T,P):
         if len(b) == 1:
             return np.array([b[0]/A[0,0]])
         
-        con = np.linalg.cond(A) #this matrix can be very ill-conditioned so we enhance precision accordingly
-        mp.dps = 30+int(np.log10(con))
-        Amp = mp.matrix(A.tolist())
-        bmp = mp.matrix(b.tolist())
-        
-        c = mp.qr_solve(Amp,bmp)
-        
-        c = np.array(list(c[0]))
-         
-        if any(c<=0.0):
-            c, rnorm = opt.nnls(A,b)
+
+        con = np.linalg.cond(A)
+
+        try:
```

Comment:
I think the error message and comment are a bit confusing: So if it is ill-conditioned we raise an error, but if it is "very ill-conditioned" we troubleshoot?

Reply:
I've adjusted this to use an if statement rather than the first try except, which makes it a bit clearer.  In general if we expect we can handle the matrix we do it normally, otherwise we fall back to arbitrary precision arithmetic, and lastly if we can't manage that we just assume all the isomers that aren't reactant channels have zero concentration which makes the calculation trivial.